Pansynchro.Protocol/BinaryDecoder.cs

```diff
@@ -222,7 +223,7 @@ private async IAsyncEnumerable<DataStream> ReadStreams(int length, DataDictionar
 				yield return new DataStream(streamName, StreamSettings.None, new StreamingReader(_reader, schema, dict, mode, () => locked = false));
 			}
 			CheckEqual(_reader.ReadByte(), (byte)Markers.End, "Expected end of stream not found.");
-			Console.WriteLine($"Bytes read: {_meter.TotalBytesRead.ToString("N0")}");
+            EventLog.Instance.AddInformationEvent($"Bytes read: {_meter.TotalBytesRead.ToString("N0")}");
```

Comment:
Check the indentation here.  This is admittedly not perfectly consistent, but I try to indent with tabs, one tab per indentation level.